pages/apis/graphql/graphql_cookbook.md

```diff
@@ -678,6 +678,23 @@ mutation UpdateSessionDuration {
 }
 ```
 
+### Update the revoke inactive tokens after
+
+You can control how many days to revoke inactive API access tokens. By default that's `NEVER` revoke inactive API access tokens, but you can set to 30, 60, 90, 180, 365 days. This is only available to Enterprise customers.
+
+```graphql
+mutation UpdateRevokeInactiveTokenAfter {
+  organizationRevokeInactiveTokensAfterUpdate(input: {
```

Comment:
Is it too late to update the name of this? I'd go with `organizationRevokeInactiveTokenPeriodUpdate` orrr if it's not an issue to remove "organization" then `revokeInactiveTokenPeriodUpdate`.

Reply:
It is following the convention ([internal link](https://github.com/buildkite/buildkite/tree/main/app/models/api/graph/mutations)), so we may need to revisit the mutation naming if we really want to rename them!

Reply:
Hmmmm I might be missing the convention you mean, but it looks like the current name doesn't match the others. The main convention I see is:

> [thing] + [action]

- clusterAgentToken + Update
- jobTypeCommand + Cancel
- pipelineSchedule + Delete

I see there are some exceptions, but most look to follow that pattern.

"organizationRevokeInactiveTokensAfter" is not a thing. Changing that part to "organizationRevokeInactiveTokensPeriod" would make it a thing and follow the convention.

Does that make sense?

Reply:
> [thing] + [action]

Yup.

`OrganizationRevokeInactiveTokensAfter` + `UpdateMutation` same as
`OrganizationAPIIPAllowlist` + `UpdateMutation`. 

They all started with `Organization[thing]` + `action`.


The field is not called `Period` [on UI](https://buildkite.com/organizations/~/security), so was keeping the same as the UI. If you strongly feel we should rename all to Period (which is possible), could you raise an escalation on Foundation team? Thanks!

Reply:
Got it, thanks! I'll raise an escalation to see how others feel. 😊